src/components/app/Home.js

```diff
@@ -384,45 +316,22 @@ class HomeImpl extends React.PureComponent<HomeProps, HomeState> {
                 profiler.firefox.com.
               </Localized>
             </p>
-          </div>
-          {/* end of grid container */}
-        </div>
-      </InstructionTransition>
-    );
-  }
-
-  _renderInstallAddonInstructions() {
-    return (
-      <InstructionTransition key={0}>
-        <div
-          className="homeInstructions"
-          data-testid="home-install-addon-instructions"
-        >
-          {/* Grid container: homeInstructions */}
-          {/* Left column: img */}
-          <img
-            className="homeSectionScreenshot"
-            src={PerfScreenshot}
-            alt="screenshot of profiler.firefox.com"
-          />
-          {/* Right column: instructions */}
-          <div>
-            <InstallButton
-              name="Gecko Profiler"
-              className="homeSectionButton"
-              xpiUrl={ADDON_URL}
-            >
-              <span className="homeSectionPlus">+</span>
-              <Localized id="Home--addon-button">Install add-on</Localized>
-            </InstallButton>
-            <DocsButton />
-            <p>
-              <Localized id="Home--addon-button-instructions">
-                Install the Gecko Profiler Add-on to start recording a
-                performance profile in Firefox, then analyze it and share it
-                with profiler.firefox.com.
+            {webChannelAvailable ? null : (
+              <Localized
+                id="Home--web-channel-unavailable"
+                elems={{
+                  code: <code />,
+                }}
+              >
+                <p data-testid="home-web-channel-unavailable-message">
+                  This profiler instance was unable to connect to the
+                  WebChannel. This usually means that it&apos;s running on a
```

Comment:
You can use `’` instead of `&apos;`, which is typographically more correct.
(but not a big deal here since it's replaced by the localization anyway)